src/object-properties.js

```diff
@@ -1,29 +1,32 @@
 /**
- * @extends {qae.GenericObjectProperties}
+ * @extends {GenericObjectProperties}
  * @entry
  */
 const properties = {
   /**
-   * Current version of this generic object definition.
+   * Current version of this generic object definition
    * @type {string}
    * @default
    */
   version: process.env.PACKAGE_VERSION,
   /**
-   * @extends {qae.HyperCubeDef}
+   * Extends HyperCubeDef, see Engine API: HyperCubeDef
+   * @extends {HyperCubeDef}
    */
   qHyperCubeDef: {
     /** @type {DimensionProperties[]} */
     qDimensions: [],
     /** @type {MeasureProperties[]} */
     qMeasures: [],
+    /** @type {schemasNxHypercubeMode} */
     qMode: 'S',
     /** @type {boolean} */
     qSuppressZero: false,
     /** @type {boolean} */
     qSuppressMissing: true,
-    /** @type {Array} */
+    /** @type {Array<integer<int32>>} */
```

Comment:
isn't int enough?

Reply:
I copied from somewhere else without thinking about it. Let me fix it.

Reply:
I mean why int32? Oddly specific for being js

Reply:
aha, how about `Array<number>`? it is the way used in typescript.

Reply:
I usually use `number[]` in jsdoc

Reply:
aha, jsdoc3. https://jsdoc.app/tags-type.htm
<img width="1269" alt="Screenshot 2022-02-23 at 10 12 52" src="https://user-images.githubusercontent.com/4471489/155289914-e0c3c115-2306-4519-b6fe-8e19e3356cb0.png">
There is a dot between Array and <MyClass> which looks odd. I gonna use `number[]`.

Reply:
Done